arangod/Graph/PathManagement/PathValidator.h

```diff
@@ -63,11 +64,12 @@ class PathValidator {
   using ProviderImpl = Provider;
   using PathStoreImpl = PathStore;
 
-  PathValidator(Provider& provider, PathStore& store,
-                PathValidatorOptions opts);
-  ~PathValidator();
+  PathValidator(Provider& provider, PathStore& store, PathValidatorOptions opts,
+                bool isSatelliteLeader);
+  virtual ~PathValidator();
```

Comment:
Who extends this class?
```suggestion
~PathValidator();
```

Reply:
Removed.